packages/layer-selector/package.json

```diff
@@ -2,10 +2,16 @@
   "name": "@agrc/layer-selector",
   "version": "4.0.0",
   "private": false,
+  "license": "MIT",
+  "homepage": "https://github.com/agrc-widgets/kitchen-sink/tree/master/packages/helpers",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/agrc/kitchen-sink.git"
+  },
   "main": "./dist/LayerSelector.js",
   "files": [
     "/dist",
-    "/package.json"
+    "layer-selector.gif"
```

Comment:
**question**: does this make the image go into the npm package? I don't think that is necessary if that is the purpose.